backend/ventas_servicio/gestor_ventas/aplicacion/lecturas/consultar_historico_ventas.py

```
@@ -0,0 +1,41 @@
+                    "total": total,
+                    "datos_mensuales": datos_mensuales
+                },
+                "historico_ventas_fallback": {
+                    "total": total_fallback,
+                    "datos_mensuales": datos_mensuales_fallback
+                }
+            }, 200
+
+    except Exception as e:
+        logging.error(f"Error al consultar reporte historico de ventas: {str(e)}")
+        return jsonify({"message": f"Error al consultar reporte historico de ventas. Intente mas tarde."}), 500
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)